src/sections/Projects/Sistent/sistent.style.js

```diff
@@ -628,6 +650,16 @@ const SistentWrapper = styled.div`
     padding-top: 1rem;
   }
 
+  /* Guidance list styling */
```

Comment:
please ensure its not effecting other guidance styles but being used consistently